portafly/portafly_api.yml

```diff
@@ -18,6 +18,27 @@ paths:
           $ref: '#/components/responses/developerAccountList'
       operationId: developerAccountList
       summary: Get developer account list
+    patch:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/DeveloperAccount'
+            examples:
+              Patch the state of 2 Dev Accounts:
+                value:
+                - id: 37
+                  state: active
+                - id: 18
+                  state: active
+        required: true
+      responses:
+        "204":
```

Comment:
If the procedure is async (which I believe it is), it probably should return `200`.

Reply:
Hmm I would be quick to agree about the send email endpoint which would trigger a worker... But this update I'm not so sure. WDYT @hallelujah @guicassolato ?

Reply:
Well I can't really answer because of the unclear summary but if it is an update then it should return 200 with some content. I really doubt about 204 no content.

Reply:
Usually, when you do a PATCH request, the successful response code is 204. More info: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH

Reply:
Not true. They just use an example and even explained it:

> A successful response is indicated by any 2xx status code.
>
> In the example below a 204 response code is used, because the response does not carry a payload body. A 200 response could have contained a payload body.

Reply:
If we go with optimism, we update everything locally and return 204, this is more React-ty because the update is almost immediate. If we wanna make it super reliable (but also slower and heavier) we return 200 with the updated collection.

Reply:
React does not have to take into account the body of the response if you do not want but an API has the responsibility of covering all use case. And for sure not sending a response is not the good way to do it

Reply:
API should not be aware of how it is used. Your React component decide to use or not the Body.

If we want a better design, React should send a Header to tell the server no need of the body: e.g. `Accept: none` BUT I do not know if this exist in HTTP spec